libraries/rush-lib/src/api/EnvironmentConfiguration.ts

```diff
@@ -373,6 +391,12 @@ export class EnvironmentConfiguration {
             break;
           }
 
+          case EnvironmentVariableNames.RUSH_PNPM_VERIFY_STORE_INTEGRITY: {
+            EnvironmentConfiguration._pnpmVerifyStoreIntegrity =
+              value === '1' ? true : value === '0' ? false : undefined;
```

Comment:
Should we just switch on whether or not it's defined?

Reply:
This is technically an override for whatever is set in .npmrc, which defaults to `true`. This means I need to be able to set it to `false`, and there is utility to setting it to `true` if your .npmrc sets it to `false` but you don't trust the environment you are running on (or know that you have tampered with the store and want to fix it).